### PR TITLE
Add separation dependency of files

### PR DESCRIPTION
**Pull request by RootenberG**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 Django~=2.2.2
+pre-commit==1.17.0
```
Comment (webknjaz) [inline requirements.txt]: Don't mix up runtime and test dependencies in one giant requirements file. Separate those.

File: .travis.yml [hunk 2]
```diff
@@ -2,5 +2,5 @@ language: python
 python: 3.7
 dist: xenial
 cache: pip
-before_install: pip install -U pre-commit
+before_install: pip install -r sep-requirements.txt
```
Comment (webknjaz) [inline .travis.yml]: What is "sep"?

Comment (RootenberG) [reply .travis.yml]: Separation.

Comment (RootenberG) [reply .travis.yml]: However, it is possible to name the cheks-requirements.txt or test-requirements.txt.

Comment (webknjaz) [reply .travis.yml]: It should be `test`, separation doesn't make sense. Are you going to call other files separation1, separation2, separation6?

File: sep-requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+pre-commit==1.17.0
```
Comment (webknjaz) [inline sep-requirements.txt]: Usually, you want to include runtime dependencies too.

File: .travis.yml (hunk 2 above)
Comment (RootenberG) [reply .travis.yml]: Ok, thanks.

File: sep-requirements.txt (hunk 3 above)
Comment (RootenberG) [reply sep-requirements.txt]: What do you mean?

Comment (RootenberG) [reply sep-requirements.txt]: Sorry, I did not quite understand.

Comment (webknjaz) [reply sep-requirements.txt]: You have Django as a runtime dependency. Usually, tests require such deps too. But I guess this can be omitted for now.

**Review by webknjaz — APPROVED** [review]

**Pull request merged**